maven-repository-manager/pom.xml

```diff
@@ -47,6 +47,10 @@
       <artifactId>spi</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.jboss.logging</groupId>
+      <artifactId>jboss-logging</artifactId>
+    </dependency>
+    <dependency>
```

Comment:
I've only included that to get the embedded Indy instance to run during tests. It's needed by Infinispan (I actually use logback + slf4j-api in Indy itself)